prime-dotnet-webapi/Services/OrganizationService.cs

```diff
@@ -287,7 +301,15 @@ private IQueryable<Organization> GetBaseOrganizationQuery()
                     .ThenInclude(a => a.SignedAgreement)
                 .Include(o => o.SigningAuthority)
                     .ThenInclude(sa => sa.Addresses)
-                        .ThenInclude(pa => pa.Address);
+                        .ThenInclude(pa => pa.Address)
+                .Include(o => o.Claims);
+        }
+
+        public async Task<bool> SwitchSigningAuthorityAsync(int organizationId, int newSigningAuthorityId)
+        {
+            var organization = await _context.Organizations.Where(o => o.Id == organizationId).SingleAsync();
```

Comment:
Because this pattern of doing `.Where(predicate)` followed by an execution statement is so common, these LINQ methods
```C#
AnyAsync()
FirstAsync()
FirstOrDefaultAsync()
SingleAsync()
SingleOrDefaultAsync()
```

all have an overloaded version that accepts the filtering predicate to simplify the code. So this line can be
```C#
var organization = await _context.Organizations
    .SingleAsync(o => o.Id == organizationId);
```